components/dot_menu/dot_menu.tsx

```diff
@@ -399,6 +399,22 @@ export class DotMenuClass extends React.PureComponent<Props, State> {
         });
     }
 
+    shortcutKey = (key: string) => {
+        return (
+            <span
+                className={classNames([
+                    'MenuItem__opacity',
+                    'MenuItem__right-decorator',
+                    {
+                        'MenuItem__text-color': !true,
+                    },
+                ])}
+            >
+                {key}
+            </span>
+        );
+    };
```

Comment:
Should this just be a part of the `Menu.ActionItem` component?

Reply:
Here all those options that are passed in as rightDecorator components:
<img width="34" alt="image" src="https://user-images.githubusercontent.com/7575921/160146798-4bce3717-ea36-4586-a89f-2451e6227d4a.png">
<img width="276" alt="image" src="https://user-images.githubusercontent.com/7575921/160146854-dfb756f5-bd42-4f89-892a-ad8fd5e9711d.png">
<img width="276" alt="image" src="https://user-images.githubusercontent.com/7575921/160146899-0e0e8310-8585-4f0e-b359-9056897581f3.png">

Reply:
0/5, just thought if it was going to be used in most cases might make sense to move it down to the lower level component

Reply:
I opened a new comment on the whole block for my thoughts on this.